source/tutorials/nix-language.md

```diff
@@ -848,6 +848,55 @@ in
 
 <!-- TODO: link to escaping rules -->
 
+(indented-strings)=
+### Indented strings
+
+Also known as “multi-line strings”.
+
+The Nix language offers convenience syntax for character strings which span multiple lines that have common indentation.
+
+Indented strings are denoted by *double single quotes* (`'' ''`).
+
+Example:
+
+```{code-block} nix
+:class: expression
+''
+multi
+line
+string
+''
+```
+
+```{code-block}
+:class: value
+"multi\nline\nstring\n"
+```
+
+Equal amounts of prepended white space are trimmed from the result.
+
+Example:
+
+```{code-block} nix
+:class: expression
+''
+  one
+   two
+    three
+''
+```
+
+```{code-block}
+:class: value
+"one\n two\n  three\n"
+```
+
+:::{note}
+Indented strings also support [string interpolation](string-interpolation). See the section on Nix's [`string`](https://nix.dev/manual/nix/stable/language/values#type-string) type in the [Nix manual](https://nix.dev/manual/nix/stable/).
+:::
+
+<!-- TODO: See [escaping rules](). -->
+
```

Comment:
```suggestion
```

The relevant content has been added, and happens to live exactly under the new link.